avm/res/desktop-virtualization/scaling-plan/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,202 @@
+targetScope = 'subscription'
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.sp-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dvspmax'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+var varHostPoolReferences = [
```

Comment:
Both variables should be part of the test invocation below. If they're not, they're not rendered in the `usageExamples` in the readme. Also - variables usually make sense to add for reuse - yet here they're only used in one location (the test below). Please move both of them and regenerate the readme when all changes are done :)